tests/roxctl/bats-tests/local/roxctl-helm-output-central-development.bats

```diff
@@ -35,7 +35,7 @@ teardown() {
 @test "roxctl-development helm output help-text should state default value for --image-defaults flag" {
   run roxctl-development helm output central-services -h
   assert_success
-  assert_line --regexp "--image-defaults.*\(development_build, rhacs, opensource\).*default \"development_build\""
+  assert_line --regexp "--image-defaults='development_build'"
```

Comment:
Why this change?

Reply:
The formatting has changed: the allowed values are printed on the next line. The test is only for the default value, hence the change.

Old:
```
  --image-defaults string   Default container images settings (development_build, rhacs, opensource); it controls repositories from where to download the images, image names and tags format (default "development_build")
```
New:
```
    --image-defaults='development_build':
        Default container images settings (development_build, rhacs, opensource); it controls repositories from where to download the 
        images, image names and tags format
```